validator/specs/lib/validateBatchTableSpec.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+var batchTableSchema = require('../data/schema/batchTable.schema.json');
+var validateBatchTable = require('../../lib/validateBatchTable');
+
+describe('batch table', function() {
+    it('validates a batch table JSON matches schema', function() {
+        var batchTableJSON = createValidBatchTableJSON();
+        expect(validateBatchTable(batchTableSchema, batchTableJSON).validation).toBe(true);
+    });
+
+    it('validates a batch table JSON object matches schema with valid binary body', function() {
+        var batchTableJSON = createValidBatchTableJSONBinary();
+        var batchTableBinary = createValidBatchTableBinary();
+        expect(validateBatchTable(batchTableSchema, batchTableJSON, batchTableBinary).validation).toBe(true);
+    });
+
+    it('returns false if batch table JSON does not match schema', function() {
+        var batchTableJSON = createInvalidBatchTableJSON();
+        expect(validateBatchTable(batchTableSchema, batchTableJSON).validation).toBe(false);
+    });
+
+    it('returns false if batch table JSON does not match schema with binary body', function() {
+        var batchTableJSON = createInvalidBatchTableJSONBinary();
+        var batchTableBinary = createValidBatchTableBinary();
+        expect(validateBatchTable(batchTableSchema, batchTableJSON, batchTableBinary).validation).toBe(false);
+    });
+
+    it('byteoffset out of bounds', function() {
```

Comment:
Make these two slightly more descriptive - `Binary property byteOffset out of bounds`